specification/marketplacecatalog/Reviews.Management/reviews.models.tsp

```diff
@@ -30,6 +32,30 @@ model UserHasReviewProperties {
   updatedAt: string;
 }
 
+@doc("Review email notification request.")
+@added(Versions.v2025_05_01)
+model ReviewEmail is ProxyResource<ReviewEmailProperties> {
```

Comment:
>ProxyResource [](http://example.com/codeflow?start=21&length=13)

How is it being a proxy resource helping ? Is this needed?

Looks like you are using only ReviewEmailProperties fields.